.eslintrc.js

```diff
@@ -6,6 +6,9 @@ module.exports = {
     es6: true,
     mocha: true
   },
+  parserOptions: {
```

Comment:
should be ``` "parserOptions": {
        "sourceType": "module",
    }```

Reply:
I think my auto formatter in vscode changed this. Let me work on it